README.md

```diff
@@ -6,9 +6,11 @@
 [![MIT License][license-badge]][license]
 [![Lean Core Badge][lean-core-badge]][lean-core-issue]
 
-ProgressBar Component for iOS, based on UIProgressView.
+ProgressBar Component for iOS and Windows, IOS based on UIProgressView.
```

Comment:
typo - 'IOS' again after Windows,
Also, is UIProgressView an iOS-only thing?

Reply:
consider:

```suggestion
ProgressBar Component for iOS (based on UIProgressView) and Windows
```